ADAL/src/request/ADAcquireTokenSilentHandler.m

```diff
@@ -152,7 +152,7 @@ - (void)acquireTokenByRefreshToken:(NSString*)refreshToken
          resultItem.authority = [_requestParams authority];
          
          
-         ADAuthenticationResult *result = [resultItem processTokenResponse:response fromRefresh:YES requestCorrelationId:_requestParams.correlationId];
+         ADAuthenticationResult *result = [resultItem processTokenResponse:response fromRefreshToken:(cacheItem ? cacheItem : nil) requestCorrelationId:_requestParams.correlationId];
```

Comment:
nit: should we simply pass `cacheItem` rather than `(cacheItem ? cacheItem : nil)`? :)